__tests__/order/order.query.test.js

```diff
@@ -137,15 +143,13 @@ describe('Order queries', () => {
         firstName: 'Test',
         phoneNumber: '380953544271',
         patronymicName: 'Test',
```

Comment:
I think it is better to move `Pattern` data to a variable file

Reply:
Fixed